src/db/index.js

```diff
@@ -14,7 +14,7 @@ const db = {
   Blocks: undefined,
   Transactions: undefined,
 };
-const MIGRATION_REGEX = /(migration)(\d+)/g;
```

Comment:
So if we are looking for multiple occurrences of that pattern among a string, `g` would be a must. In our case we just checking if the whole string follows the pattern, `g` is not necessary.

But more importantly, by having `g`, there exists an inner index counter for the purpose of multiple matches. But since we declare it as global, the counter won't have a chance to reset, which affects further searches in a loop.

We could still have `g`, but we need to put the declaration inside the loop. I removed it because it's an overkill.

Reply:
Gotcha. Ya I remember reading about this before.